Code/PgSQL/rdkit/expected/btree.out

```diff
@@ -200,14 +200,14 @@ SELECT count(*) FROM pgsfp WHERE
 	f < morgan_fp('C1C(OC2=CC(=CC(=C2C1=O)O)O)C3=CC(=C(C=C3)O)O'::mol, 1);
  count 
 -------
-    54
+    55
```

Comment:
It's a reasonable question, but the important piece for these tests are not the results themselves, but that the answers returned by the searches done with and without the index are the same.